<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogPanel</class>
 <widget class="QWidget" name="LogPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Level</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_logLevel">
       <item>
        <property name="text">
         <string>Error</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Debug</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>warning</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Info</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_saveBtn">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="m_eraseBtn">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="m_logview">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="m_eraseAllAct">
   <property name="text">
    <string>Erase All</string>
   </property>
  </action>
  <action name="m_saveAct">
   <property name="icon">
    <iconset theme="QStyle::SP_DriveHDIcon"/>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Provide this file to Rolisteam developers</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_eraseAllAct</sender>
   <signal>triggered()</signal>
   <receiver>m_logview</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
